lib/puppet/pops/serialization/json.rb

```diff
@@ -248,14 +248,14 @@ def initialize(io)
 
     def re_initialize(io)
       parsed = parse_io(io)
-      raise SerializationError, "JSON stream is not an array. It is a #{io.class.name}" unless parsed.is_a?(Array)
+      raise SerializationError, _("JSON stream is not an array. It is a %{klass}") % { klass: io.class.name } unless parsed.is_a?(Array)
```

Comment:
@Magisus JSON is a keyword that refers to a language, should it be left out of translation?